packages/api-client/src/components/ApiClient/Request/Request.vue

```diff
@@ -92,318 +92,321 @@ const readOnly = true
-  z-index: 10;
-}
-.input:first-of-type {
-  border-radius: var(--scalar-radius) var(--scalar-radius) 0 0;
-}
-.input:first-child:last-child {
-  border-radius: var(--scalar-radius);
-}
-.input:last-child {
-  box-shadow: none;
-  border-radius: var(--scalar-radius);
-}
-.input__half:first-of-type {
-  border-radius: var(--scalar-radius) 0 0 0;
-}
-.input__half:nth-of-type(2) {
-  border-radius: 0 var(--scalar-radius) 0 0;
-}
-.authentication-form {
-  box-shadow: 0 0 0 1px var(--scalar-border-color);
-  border-radius: var(--scalar-radius);
-  width: 100%;
-  display: flex;
-  flex-flow: wrap;
-}
-.input__half {
-  width: 50%;
-}
-.input__half + .input__half {
-  border-left: 1px solid var(--scalar-border-color);
-}
-.input__half:focus-within {
-  border-color: transparent;
-}
-.input label,
-.input input {
-  padding: 9px;
-  border: 0;
-  outline: none;
-  font-size: var(--scalar-micro);
-  color: var(--scalar-color-2);
-  width: 100%;
-  background: transparent;
-  appearance: none;
-  -webkit-appearance: none;
-  left: 0;
-}
-.input label {
-  color: var(--scalar-color-1);
-  width: fit-content;
-  padding-right: 0;
-  white-space: nowrap;
-  cursor: text;
-}
-.input input {
-  position: relative;
-  z-index: 99;
-}
-.input input:not(:placeholder-shown) + label {
-  color: var(--scalar-color-2);
-}
-.select {
-  background: var(--scalar-background-1);
-  border-radius: var(--scalar-radius-lg);
-  font-size: var(--scalar-micro);
-  border: 1px solid var(--scalar-border-color);
-  width: 100%;
-  position: relative;
-  margin-bottom: 6px;
-}
-.select:focus-within {
-  background: var(--scalar-background-3);
-}
-.select:hover {
-  background: var(--scalar-background-3);
-}
-.select svg {
-  position: absolute;
-  right: 9px;
-  pointer-events: none;
-  color: var(--scalar-color-2);
-  width: 6px;
-  top: 10px;
-}
-.select label {
-  display: block;
-  font-size: 10px;
-  color: var(--scalar-color-2);
-  position: absolute;
-  left: 9px;
-  top: 6px;
-}
-.select select {
-  background: transparent;
-  outline: none;
-  border: none;
-  -moz-appearance: none;
-  -webkit-appearance: none;
-  appearance: none;
-  font-size: var(--scalar-micro);
-  color: var(--scalar-color-1);
-  width: 100%;
-  padding: 14px 9px 4px 9px;
-  top: 0;
-  position: relative;
-  cursor: pointer;
-}
 
-.check {
-  display: flex;
-  position: relative;
-  cursor: pointer;
-  align-items: center;
-  font-size: var(--scalar-micro);
-  padding: 6px 9px;
-  border-radius: var(--scalar-radius);
-  user-select: none;
-  width: 100%;
-  outline: none;
-}
-.check:focus-within {
-  box-shadow: 0 0 0 1px var(--scalar-color-1);
-}
-.checkmark:hover {
-  background: var(--scalar-background-3);
-}
-.check:focus-within {
-  border-color: var(--scalar-color-1);
-}
-.check p {
-  color: var(--scalar-color-3);
-}
-.check input {
-  position: absolute;
-  opacity: 0;
-  cursor: pointer;
-  height: 0;
-  width: 0;
-}
+.scalar-api-client__main__left {
```

Comment:
I think since were using nesting here we might need to set `lang="postcss"` on the `<style>` block?

Reply:
Done!